README.md

```diff
@@ -149,7 +149,14 @@ after you update some code in order to see changes without having to rebuild all
 ### Running Docker Compose on a Remote Machine
 To run Docker Compose on a remote machine, modify the `hostname` field in `fence-config.yaml`, `peregrine_creds.json`, and `sheepdog_creds.json` in the `Secrets` directory.
 
-* * *
+### Dumping config files and logs
+
+If you are encountering difficulties while setting up Docker Compose and need help from the Gen3 team, you can use the `dump.sh` script to create a zip file of your configuration and current logs, which you can share to get help.
```

Comment:
should add a note that the configurations can contain *sensitive* credentials maybe?

Reply:
mm yeah 👍